### PR TITLE
Add Dumping command

### PR DESCRIPTION
**Pull request by JaffaCat**
This is a redo of PR 165, as that PR had several issues previously which should be fixed in this PR. 

This PR adds the .dumping command to Citra's Discord bot. By default anyone can use this command to inform users on what the process of dumping a game is.

_Suggested by SleepingSnake from the Citra Discord server._

### CODE REVIEW

File: citra.json
Comment (FearlessTobi) [inline citra.json:1]: why is this file added?

**Review by FearlessTobi — CHANGES_REQUESTED** [review]

File: citra.json
Comment (JaffaCat) [reply citra.json:1]: Because it needs to add the dumping command?

Comment (sakuramipha) [reply citra.json:1]: Yes, it's the same as last time. You need to delete that file. It doesn't belong there, the only `citra.json` file you need is in `src/responses`

Comment (JaffaCat) [reply citra.json:1]: Should be fixed now

Comment (sakuramipha) [reply citra.json:1]: No, you deleted the wrong one. We want to keep `src/responses/citra.json` and delete `citra.json`

Comment (JaffaCat) [reply citra.json:1]: Hopefully that was it. I cannot believe that there was a second citra.json in the code. That was such a simple fix

File: src/responses/citra.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
...
       "roms": { "reply": "Please read our __community rules__. Warez/downloading games talk is strictly prohibited. To prevent legal issues, you are not allowed to post links or refer to any kind of ROM, NAND, ISO, game, or other copyrighted material that has been illegally obtained or shared. <https://citra-emu.org/rules/>"},
       "dump-game": { "reply": "Please refer to our __game dumping guides__. \nFor Cartridges: <https://citra-emu.org/wiki/dumping-game-cartridges/> \nFor Installed Titles: <https://citra-emu.org/wiki/dumping-installed-titles/> \nTo dump DLC and Updates, use this guide <https://citra-emu.org/wiki/dumping-updates-and-dlcs/> and install the results through `File -> Install CIA...`"},
+      "dumping": {"reply": "Dumping is the act of copying a game from your system/gamecart to an SD card. \nDumping is legal, however sharing the dumps with anyone else is piracy and thus is illegal."},
```
Comment (sakuramipha) [inline src/responses/citra.json:8]:
```suggestion
      "dumping": {"reply": "Dumping is the act of creating an archival copy of your own legally purchased games. This process requires a hacked console to perform the dump \nDumping is legal, however sharing your dumped archival copies with anyone else is illegal."},
```
Clarifies on what dumping does, adds section about needing a console, clarifies what "dump" is when talking about the legality.

Comment (JaffaCat) [reply src/responses/citra.json:8]: Will change it later, I did originally have it set to something like that, but must have changed it unintentionally.

**Review by sakuramipha — APPROVED** [review]

**Pull request merged**